.pipelines/Bicep.yml

```diff
@@ -67,7 +67,7 @@ steps:
     command: 'publish'
     publishWebProjects: false
     projects: ./src/Bicep.Cli/Bicep.Cli.csproj
-    arguments: '--configuration $(BuildConfiguration) --self-contained true -p:PublishTrimmed=true -p:PublishSingleFile=true -r ${{ parameters.rid }} /p:PublicRelease=${{ parameters.official }}'
+    arguments: '--configuration $(BuildConfiguration) --self-contained true -p:PublishTrimmed=true -p:PublishSingleFile=true -p:TrimmerDefaultAction=copyused -p:SuppressTrimAnalysisWarnings=true -r ${{ parameters.rid }} /p:PublicRelease=${{ parameters.official }}'
```

Comment:
>-p:TrimmerDefaultAction=copyused -p:SuppressTrimAnalysisWarnings=true [](http://example.com/codeflow?start=125&length=69)

Need eddifferent trimmer settings for WASM vs the CLI, so the simplest option seemed to be configuring it in the CI jobs. I will explain more on trimming in another comment elsewhere.